src/Widget/Providers/WidgetServiceProvider.php

```diff
@@ -38,10 +38,10 @@ public function boot()
     protected function registerConfig()
     {
         $this->publishes([
-            __DIR__ . '/../Config/widget.php' => config_path('widget.php'),
+            __DIR__.'/../Config/widget.php' => config_path('widget.php'),
         ]);
         $this->mergeConfigFrom(
-            __DIR__ . '/../Config/widget.php', 'widget'
+            __DIR__.'/../Config/widget.php', 'widget'
```

Comment:
__[code_sniffer]__
* Only one argument is allowed per line in a multi-line function call

<!-- Comment by [SideCI](https://sideci.com) -->